generatedDockerfiles/Dockerfile.centos7-devel

```diff
@@ -180,6 +180,11 @@ RUN cd ${RAPIDS_DIR}/rmm && \
 RUN cd ${RAPIDS_DIR}/cudf && \
     source activate rapids && \
     ./build.sh
+RUN cd ${RAPIDS_DIR}/cuspatial && \
+    source activate rapids && \
+    export CUSPATIAL_HOME="$PWD" && \
```

Comment:
Should I do export here or maybe use the generated build script similar to xgboost?

Reply:
This is fine for now. Can you also put a FIXME comment saying that we shouldn't need to set these vars long-term?

Reply:
Fixed